### PR TITLE
allow runFile to specify hashes

### PR DESCRIPTION
**Pull request by haseebrabbani**
updating the `run --file` command to allow specifying block/tx/alert hashes (instead of just json objects), as well as sequences. this will help Christian test sequences of events that are too long to specify on the commandline

### CODE REVIEW

File: cli/commands/run/run.file.ts [hunk 1]
```diff
@@ -18,29 +30,47 @@ export function provideRunFile(
...
       console.log('running block events...')
       for (const blockEvent of blockEvents) {
-        const findings = await handleBlock(blockEvent)
-        console.log(`${findings.length} findings for block ${blockEvent.hash} ${findings}`)
+        if (typeof blockEvent === 'string' || typeof blockEvent === 'number') {
+          await runHandlersOnBlock(blockEvent)
+        } else {
+          const findings = await handleBlock(blockEvent)
+          console.log(`${findings.length} findings for block ${blockEvent.hash} ${findings}`)
+        }
```
Comment (Robert-H-Leonard) [inline cli/commands/run/run.file.ts:43]: What block events could have a type that isn't a string or a number?

File: cli/commands/run/run.file.ts [hunk 2]
```diff
@@ -18,29 +30,47 @@ export function provideRunFile(
...
     if (handleTransaction && transactionEvents?.length) {
       console.log('running transaction events...')
       for (const transactionEvent of transactionEvents) {
-        const findings = await handleTransaction(transactionEvent)
-        console.log(`${findings.length} findings for transaction ${transactionEvent.transaction.hash} ${findings}`)
+        if (typeof transactionEvent === 'string') {
+          await runHandlersOnTransaction(transactionEvent)
+        } else {
+          const findings = await handleTransaction(transactionEvent)
+          console.log(`${findings.length} findings for transaction ${transactionEvent.transaction.hash} ${findings}`)
```
Comment (Robert-H-Leonard) [inline cli/commands/run/run.file.ts:54]: Same as question above?

File: cli/commands/run/run.file.ts (hunk 1 above)
Comment (haseebrabbani) [reply cli/commands/run/run.file.ts:43]: the file option allows specifying a BlockEvent json object

File: cli/commands/run/run.file.ts (hunk 2 above)
Comment (haseebrabbani) [reply cli/commands/run/run.file.ts:54]: a TransactionEvent json object

**Comment by haseebrabbani** [comment]
example json file:
```
{
  "sequenceEvents": [
    "0x856354a9c758ae9c2af05c2e34d780db724c5fb6821df42aa4b88b9211bd261e",
    "tx0xb9b6feddd9a85055be80404786b92c12930bb391113023b0365383951b79544f",
    "0x00b733e8fb2d4470384440d533f9f58cd4b92bb32933ab214eef237e49093d5f",
    "16339207",
    "tx0x7fe3ad95e798c4a8fe73e459f1fe04e56e5ddaf35185ed9ce4d0d5fa4d017eb5,16339131,0x8189a3ef9abf5f34a85f65a7da6b09a4170c481d3cc44621263ae0ad134bd85c"
  ]
}
```

**Review by Robert-H-Leonard — APPROVED** [review]

**Pull request merged**